Week01/Problem01/dskinner_01.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+#
+# Here is a solution to the general problem for multiples of 2 numbers (a, b) below a certain value. This can be imported to another script:
+#
+def multiples(a, b, value):
+    arr = np.arange(value)
+    num = []
+    for i in arr:
+        if (((i/a).is_integer()) == True) or (((i/b).is_integer()) == True):
+            num.append(int(i))
+        else:
+            continue
+    return 'The sum of all the multiples of ' + str(a) + ' and ' + str(b) + ' is ' + str(np.sum(num)) +'!'
+
+#
+# Alternatively, here is the direct solution which will print upon typing `python dskinner_01.py`
```

Comment:
If ever you wish to _only_ run a particular piece of code when a script is called directly, i.e., 
```
$ python myscript.py
```
on the command line (as you have intended here), you can place the code behind a special conditional statement:
```
if __name__ == "__main__":
   # This is only executed when script is run directly from command line
   code code code
```
The special variable `__name__` is set by Python when any script is executed; it is specifically set to `"__main__ "` when the script is run directly.  This is super useful when you're writing a module full of functions, and the only reason to run the script directly is for testing purposes.

Reply:
Good idea! I didn't know about this until last weeks lecture. I will begin to implement this.